components/confirm_modal.jsx

```diff
@@ -107,7 +109,12 @@ export default class ConfirmModal extends React.Component {
 
     handleKeypress = (e) => {
         if (e.key === 'Enter' && this.props.show) {
-            this.handleConfirm();
+            const cancelButton = document.getElementById('cancelModalButton');
```

Comment:
What happens if I change focus away from both buttons -- does enter still trigger `handleConfirm`? Do our buttons support triggering `onClick` for themselves if they have focus and we press enter on them?

Reply:
Pressing enter triggers `handleConfirm` as long as the cancel button does not have focus. The listener is set up this "global" way so that if the `showCheckbox` prop is used to display an optional checkbox, the user can check the checkbox and immediately press enter to submit the modal's form.

https://github.com/mattermost/mattermost-webapp/blob/e1816712646b44dc6cae185800370c57661aa44b/components/confirm_modal.jsx#L88-L96

https://github.com/mattermost/mattermost-webapp/blob/e1816712646b44dc6cae185800370c57661aa44b/components/confirm_modal.jsx#L110-L119